joplin/base/wagtail_hooks.py

```diff
@@ -72,8 +72,8 @@ def configure_main_menu(request, menu_items):
 
 @hooks.register('register_admin_menu_item')
 def register_page_list_menu_item():
-    home = HomePage.objects.first()
-    return MenuItem('Pages', reverse('wagtailadmin_explore', args=[home.pk]), classnames='icon icon-home', order=10)
+    # home = HomePage.objects.first()
```

Comment:
so glad we don't have the hacky /3/ logic anymore

Reply:
Thats been changed for awhile